frameworks/PHP/php-laravel/setup_hhvm.sh

```diff
@@ -15,5 +15,5 @@ sed -i 's|/home/vagrant/FrameworkBenchmarks/installs/nginx/|'"${IROOT}"'/nginx/|
 
 export PATH="${PHP_HOME}/bin:${PHP_HOME}/sbin:$PATH"
 
-hhvm --config $TROOT/deploy/config.hdf --user $(whoami) -m daemon
+hhvm -m daemon --config $TROOT/deploy/config.hdf --user $(whoami)
```

Comment:
Just so we have some explanation on this SOMEWHERE: this was an annoying fix to `hhvm` simply ignoring input flags because of an incorrect order and STILL starting up but binding to completely different ports as a completely different user.

When `-m daemon` was at the end, `hhvm` would bind to 9000 as `www-user` regardless of the other flags, but moving it to the first argument (hint: not an argument, apparently) fixes this.

SO ANNOYING!!!